packages/vscode-extension/src/officeChat/commands/generatecode/generatecodeCommandHandler.ts

```diff
@@ -66,7 +67,7 @@ export default async function generatecodeCommandHandler(
     }
   }
 
-  const isHarmful = await isInputHarmful(request, token);
+  const isHarmful = await isInputHarmful(request, token, officeChatTelemetryData);
```

Comment:
Pass in the `officeChatTelemetryData.requestId`, and parse the telemetry object using `OfficeChatTelemetryData.get(requestId)`